README.md

```diff
@@ -140,3 +140,15 @@ Paste Photon License to: bin_Win64 or bin_Win32
 It might show .NET Framework 3.5 missing, which can be ignored.   
 Start PhotonController.exe.  
 The PhotonController will be started at your taskbar tray, rightclick and find UberStrok, click Start as application.   
+
+#Steps for Realtime not starting (PhotonLicense.dll Blocked).
```

Comment:
add space between # and Steps so it is formatted correctly

Reply:
Check Now